### PR TITLE
Remaps the Dwayne.

### PR DESCRIPTION
**Pull request by Sadhorizon**
## About The Pull Request
As the title might hint, this remaps the Dwayne.
**TODO:**
- [x] hydrogen thrusters (after that PR is merged)
- [x] fix bridge
- [x] fix atmos
- [x] finish the wiki page  --- not finished, but good enough for now
- [x] processing console directionals

Most glaring changes are addition of second hangar (dedicated to cargo), addition of crew areas, addition of a smelter setup, and removal of medical area. Miners have lost some gear too - namely, crusher, KA and the explorer suits are now dead.

Also fixes a bunch of weird issues Dwayne had.

But yea, this is pretty much a complete remap. Every room was tweaked pretty hard, while trying to keep the ship's soul.

scrnshot1

StrongDMM-2024-07-09 00 04 40

StrongDMM-2024-07-01 01 24 00

**JOB SLOT CHANGES:**
- No doctor - there is no medical. What do you want them to do???
- Two less deckhands - you REALLY don't need three.

## Why It's Good For The Game
Dwayne is old and can use some tweaks.

## Changelog

:cl:
tweak: Remapped the Dwayne-class Long Range Mining Transport.
add: Added a random plushie spawner and a random moth plushie spawner.
/:cl:

### CODE REVIEW

File: code/modules/clothing/outfits/factions/independent.dm [hunk 1]
```diff
@@ -540,20 +540,25 @@
 	dcoat = /obj/item/clothing/suit/hooded/wintercoat/cargo
 	shoes = /obj/item/clothing/shoes/sneakers/brown
 	glasses = /obj/item/clothing/glasses/sunglasses
-	l_hand = /obj/item/clipboard
+	r_pocket = /obj/item/clipboard
```
Comment (meemofcourse) [inline code/modules/clothing/outfits/factions/independent.dm:543]: i'm not sure if clipboards fit in pockets

File: code/modules/clothing/outfits/factions/independent.dm [hunk 2]
```diff
@@ -540,20 +540,25 @@
...
 	backpack_contents = list(/obj/item/modular_computer/tablet/preset/cargo=1)
 
 	chameleon_extras = /obj/item/stamp/qm
 
 /datum/outfit/job/independent/quartermaster/western
 	name = "Independent - Quartermaster (Western)"
 
-	uniform = /obj/item/clothing/under/rank/cargo/qm
-	suit = /obj/item/clothing/suit/toggle/hazard
+	belt = /obj/item/storage/belt/utility/full // securing stuff and minor renovations
```
Comment (meemofcourse) [inline code/modules/clothing/outfits/factions/independent.dm]:
```suggestion
	belt = null
```
map the belt in if needed, please

**Review by meemofcourse — APPROVED** [review]
not much else to add, cool remap

File: code/modules/clothing/outfits/factions/independent.dm (hunk 1 above)
Comment (Sadhorizon) [reply code/modules/clothing/outfits/factions/independent.dm:543]: They do.

**Pull request merged**